guide/qip/qip-simulator.rst

```diff
@@ -69,69 +84,74 @@ we measure each of the three qubits at the end.
   qc.add_measurement("M2", targets=[2], classical_store=2)
 
 To get all the possible output states along with the respective probability of observing the
-outputs, we can use the :func:`qutip.qip.QubitCircuit.run_statistics()` function:
+outputs, we can use the :func:`~qutip.qip.circuit.QubitCircuit.run_statistics` function:
 
 .. testcode::
 
-  result = qc.run_statistics(state=tensor(basis(2, 0), basis(2, 0), basis(2, 0)))
-  states, probabilities = result.get_results()
+    result = qc.run_statistics(state=tensor(basis(2, 0), basis(2, 0), basis(2, 0)))
```

Comment:
can also be spelled `state=basis([2, 2, 2], [0, 0, 0])` if you prefer (my first ever contribution to QuTiP!)

Reply:
Hahaha, this is Sidhant's code, I do use `state=basis([2, 2, 2], [0, 0, 0])` nowadays. But I don't know it is young. Just discovered it at some point and started to use it :）